source/guides/aggregate/adding-cosginatures-aggregate-complete.rst

```diff
@@ -0,0 +1,92 @@
+:orphan:
+
+.. post:: 13 July, 2020
+    :category: Aggregate Transaction
+    :tags: SDK, CLI
+    :excerpt: 1
+    :nocomments:
+
+#############################################################
+Adding cosignatures to aggregate complete trasactions offline
+#############################################################
+
+This guide will show you how to add cosignatures to an aggregate complete transaction while offline, then use an online device to announce the transaction.
+
+This can be useful for when you are keeping your private keys in an offline device (cold wallet) for security reasons. By signing adding the cosignatures offline, the cosignatories will be able to execute transactions from your cold wallet while keeping their private keys completely safe.
+
```

Comment:
Consider explaining that can be useful to not have to lock 10 XYM to announce aggregate bonded transactions.